### ISSUE TITLE
SyntaxError: EOL while scanning string literal

### ISSUE THREAD

**Issue by maximousblk** [soxoj/maigret#150]
![image](https://user-images.githubusercontent.com/28438021/118239394-5d2a4c00-b4b7-11eb-881c-f9aab94d84fa.png)

<details>
<summary>text</summary>

```red
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.9.5/x64/bin/maigret", line 33, in <module>
    sys.exit(load_entry_point('maigret==0.2.3', 'console_scripts', 'maigret')())
  File "/opt/hostedtoolcache/Python/3.9.5/x64/lib/python3.9/site-packages/maigret/maigret.py", line 656, in run
    loop.run_until_complete(main())
  File "/opt/hostedtoolcache/Python/3.9.5/x64/lib/python3.9/asyncio/base_events.py", line 642, in run_until_complete
    return future.result()
  File "/opt/hostedtoolcache/Python/3.9.5/x64/lib/python3.9/site-packages/maigret/maigret.py", line 581, in main
    results = await maigret(
  File "/opt/hostedtoolcache/Python/3.9.5/x64/lib/python3.9/site-packages/maigret/checking.py", line 575, in maigret
    cur_results = await executor.run(tasks_dict.values())
  File "/opt/hostedtoolcache/Python/3.9.5/x64/lib/python3.9/site-packages/maigret/executors.py", line 25, in run
    results = await self._run(tasks)
  File "/opt/hostedtoolcache/Python/3.9.5/x64/lib/python3.9/site-packages/maigret/executors.py", line 40, in _run
    return await asyncio.gather(*futures)
  File "/opt/hostedtoolcache/Python/3.9.5/x64/lib/python3.9/site-packages/maigret/checking.py", line 428, in check_site_for_username
    query_notify.update(response_result['status'], site.similar_search)
  File "/opt/hostedtoolcache/Python/3.9.5/x64/lib/python3.9/site-packages/maigret/notify.py", line 233, in update
    ids_data_text = get_dict_ascii_tree(self.result.ids_data.items(), " ")
  File "/opt/hostedtoolcache/Python/3.9.5/x64/lib/python3.9/site-packages/maigret/utils.py", line 78, in get_dict_ascii_tree
    field_value = get_dict_ascii_tree(eval(field_value), prepend_symbols)
  File "<string>", line 1
    ['maximousblk
                ^
SyntaxError: EOL while scanning string literal
Error: Process completed with exit code 1.
```

</details>

**Comment by soxoj** [comment]
Thank you, looks like an unexpected clipping of data received from socid_extractor.

Could you remember the exact command you've run?

**Comment by maximousblk** [comment]
```sh
# pip3 install git+https://github.com/soxoj/maigret.git
maigret maximousblk -fo maximousblk --no-progressbar --no-recursion --timeout 1024 -a -T -C -X -H -P -J simple
```

**Comment by soxoj** [comment]
The problem caused by Gravatar email extracting. I've made a fix in [socid_extractor](https://github.com/soxoj/socid-extractor/pull/89), please install it from git and check that the problem was fixed for you.

**Comment by maximousblk** [comment]
yes this seems to be fixed now

**Comment by soxoj** [comment]
I've updated Maigret dev version.

**Comment by maximousblk** [comment]
fixed 👍

**Comment by soxoj** [comment]
Fix released in Maigret v0.2.4.
